.github/workflows/docker.yaml

```diff
@@ -0,0 +1,116 @@
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: build
+        env:
+          image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-metad:${{ github.sha }}
+          nightly_image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-metad:nightly
+        run: |
+          docker build -t ${IMAGE_NAME} -f docker/Dockerfile.metad .
+          docker tag ${IMAGE_NAME} ${NIGHTLY_IMAGE_NAME}
+          docker push ${IMAGE_NAME}
+          docker push ${NIGHTLY_IMAGE_NAME}
+        shell: bash
+
+  storaged:
+    name: storaged
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: azure/container-actions/docker-login@master
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: build
+        env:
+          image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-storaged:${{ github.sha }}
+          nightly_image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-storaged:nightly
+        run: |
+          docker build -t ${IMAGE_NAME} -f docker/Dockerfile.storaged .
+          docker tag ${IMAGE_NAME} ${NIGHTLY_IMAGE_NAME}
+          docker push ${IMAGE_NAME}
+          docker push ${NIGHTLY_IMAGE_NAME}
+        shell: bash
+
+  graphd:
+    name: graphd
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: azure/container-actions/docker-login@master
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: build
+        env:
+          image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-graphd:${{ github.sha }}
+          nightly_image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-graphd:nightly
+        run: |
+          docker build -t ${IMAGE_NAME} -f docker/Dockerfile.graphd .
+          docker tag ${IMAGE_NAME} ${NIGHTLY_IMAGE_NAME}
+          docker push ${IMAGE_NAME}
+          docker push ${NIGHTLY_IMAGE_NAME}
+        shell: bash
+
+  console:
+    name: console
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: azure/container-actions/docker-login@master
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: build
+        env:
+          image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-console:${{ github.sha }}
+          nightly_image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-console:nightly
+        run: |
+          docker build -t ${IMAGE_NAME} -f docker/Dockerfile.console .
+          docker tag ${IMAGE_NAME} ${NIGHTLY_IMAGE_NAME}
+          docker push ${IMAGE_NAME}
+          docker push ${NIGHTLY_IMAGE_NAME}
+        shell: bash
+
+  nebula_graph:
```

Comment:
It will test all? I can't find the `Dockerfile`

Reply:
Dockerfiles in another PR #951